.chloggen/2995-openshift-dashboard.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: collector
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Create automatically an OpenShift dashboard to visualize OpenTelemetry Collector metrics
+
+# One or more tracking issues related to the change
+issues: [2995]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: To enable this feature, you need to specify the `--create-openshift-dashboard` argument to the operator
```

Comment:
Does it make sense to start openshift specific flags with an `--openshift-*` prefix?

Reply:
I think it's a nice idea :)